_troubleshooting/mst/authentication-failed.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Authentication Failed when launching SAML application
+section: troubleshooting
+products: [mst]
+topics: [authentication]
+errors:
+  - language: bash
+    message: |-
+      Authentication Failed
+      Login failed.  Please contact your account administrator for more details.
+keywords: [authentication, SAML, Okta]
+tags: [mst]
+---
+
+You might see this error when launching MST SAML application:
+
+   ```bash
+   Authentication Failed
+
+   Login failed.  Please contact your account administrator for more details.
+   ```
```

Comment:
Don't indent this as that automatically creates a codeblock, and that will make your backtick lines show in the final output.